frontend/src/components/regions/UserActions/index.jsx

```diff
@@ -6,86 +6,61 @@ import { useParams } from "react-router-dom";
 import { connect } from "react-redux";
 import { apiReduxApps } from "../../../apiAccess/apiApps";
 
-import { ACTION_NAMES } from "./constants";
 import { getLatestMatchingCall } from "@prelude/redux-api-toolbox/src/apiEntities/selectors";
 import Loading from "../../widgets/Loading";
-
-// TODO: look nice
-
-const days = [
-  "Sunday",
-  "Monday",
-  "Tuesday",
-  "Wednesday",
-  "Thursday",
-  "Friday",
-  "Saturday",
-];
+import { eventTypeColors } from "../../../colors";
+import { addEventColorsToLogEntries, matchEventTypesWithColors } from "./utils";
 
 function UserActionsUnconnected({
   authedUserId,
-  projectReviews,
-  cardSummaries,
-  fetchProjectReviewsPages,
   fetchCardCompletions,
```

Comment:
fetchCardCompletions is from the old activity log

Reply:
part of burndown